### PR TITLE
PORT-10429 - Guide to track SLO 

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description

Created a guide to track slo's and sli's using Port's new relic integration
## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- Guide (`/guides/all/track-slos-and-slis-for-services`)

### CODE REVIEW

File: docs/guides/all/track-slos-and-slis-for-services.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+This guide assumes the following:
+- You have a Port account and have completed the [onboarding process](https://docs.getport.io/quickstart).
+- You have [installed and set up Port's New Relic integration](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/apm-alerting/newrelic).
+- You have the `New Relic Service Level` blueprint configured in your Port environment.
+
+
+
+## Data Model Setup
+
+<h2> Blueprint setup </h2>
```
Comment (hadar-co) [inline docs/guides/all/track-slos-and-slis-for-services.md:28]: fix headers

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/guides/all/track-slos-and-slis-for-services.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+4. Click **Save**.
+5. Click on the **`...`** button in the top right corner of the table and select **Customize table**.
+6. Group the table by **Domain**.
+7. Apply a filter to display only the **Failed** SLOs.
+
+</details>
+
+
+
+## Visualizing SLI Trends
```
[the remark this answers is not in the document]
Comment (kodjomiles) [reply docs/guides/all/track-slos-and-slis-for-services.md:223]: @hadar-co Why are we not having a separate section for visualization but adding it to the dashboard setup
I thought they are supposed to be two different things.Mostly we have a separate section for visualization and dashboards

File: docs/guides/all/track-slos-and-slis-for-services.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+## Prerequisites
+
+This guide assumes the following:
+- You have a Port account and have completed the [onboarding process](https://docs.getport.io/quickstart).
+- You have [installed and set up Port's New Relic integration](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/apm-alerting/newrelic).
+- You have the `New Relic Service Level` blueprint configured in your Port environment.
+
+
+
+## Data Model Setup
```
[the remark this answers is not in the document]
Comment (kodjomiles) [reply docs/guides/all/track-slos-and-slis-for-services.md:25]: Most of the docs have it as Data Model Setup, I thought we should be consistent ?? or

File: docs/guides/all/track-slos-and-slis-for-services.md [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+
+In this setup, we will add or update the `New Relic Service Level` blueprint.   
+**Skip** to [update blueprint](#update-the-blueprint) if you already have the blueprint.
+
+
+### Add the blueprint
+Follow the steps below to **add** the `New Relic Service Level` blueprint:
+
+1. **Go to the [Builder](https://app.getport.io/settings/data-model) in your Port portal.**
```
[the remark this answers is not in the document]
Comment (kodjomiles) [reply docs/guides/all/track-slos-and-slis-for-services.md:34]: noted

File: docs/guides/all/track-slos-and-slis-for-services.md [hunk 5]
```diff
@@ -0,0 +1,400 @@
...
+   ```
+
+   </details>
+
+5. **Click "Save" to create the blueprint.**
+
+### Update the blueprint
+Follow the steps below to **update** the `New Relic Service Level` blueprint:
+
+1. **Navigate to the `New Relic Service Level` blueprint in your Port [Builder](https://app.getport.io/settings/data-model).**
```
[the remark this answers is not in the document]
Comment (kodjomiles) [reply docs/guides/all/track-slos-and-slis-for-services.md:126]: We should only use when we want to highlight yeah

**Review by hadar-co — APPROVED** [review]

**Pull request merged**